js/jquery.validation.js

```diff
@@ -149,48 +150,50 @@
                             });
                         },
                         error: function(jqXHR, textStatus, errorThrown) {
-                            var returnType = jqXHR.getResponseHeader('content-type');
-                            
-                            if ( returnType == "application/json; charset=utf-8" && jqXHR.status != 200 && jqXHR.status != 0 ) {
-                                // HTTP Error with JSON response
-                                var response = $.parseJSON(jqXHR.responseText)
-                                $form.validation('parseResponse', response);
-
-                                // Deprecated (replaced by valid, invalid & redirect events) - Trigger validationComplete event
-                                $form.trigger({
-                                    type: 'validationComplete',
-                                    response: response
-                                });
+                            if ( errorThrown != "abort" ) {
+                                var returnType = jqXHR.getResponseHeader('content-type');
```

Comment:
only change here is wrapping existing code in an if statement to ignore abort errors